processor/src/stack/mod.rs

```diff
@@ -134,13 +134,13 @@ impl Stack {
         result
     }
 
-    /// Returns [ProgramOutputs] consisting of all values on the stack and all addresses in the
+    /// Returns [StackOutputs] consisting of all values on the stack and all addresses in the
     /// overflow table that are required to rebuild the rows in the overflow table.
-    pub fn get_outputs(&self) -> ProgramOutputs {
+    pub fn compute_stack_outputs(&self) -> StackOutputs {
```

Comment:
Get does sound a bit ambiguous to me as in Rust we commonly remove the prefix `get_` just to keep the names of the functions shorter, but we don't re-signify it.

It's like: we commonly use `fn foo` instead of `fn get_foo`, but should be equivalent.

I would prefer `build_` over `get_` for that reason (or any other meaningful name that will disambiguate and make it clear that we are not just getting an attribute, but computing a value).

Since this is indeed used in other parts, we can keep `get_` for now, but it might be worthy to rework this in the future. Alternatively, I can just rename everywhere else to `build_` in this PR. wdyt?

Reply:
I'm fine with either `get_` or `build_` here. `build_` is probably a bit better.